lib/logic.rb

```diff
@@ -0,0 +1,69 @@
+# frozen_string_literal: true
+
+class GameLogic #:nodoc:
+  attr_accessor :number, :board, :turn, :array_of_number
+  attr_reader :tie, :token
+  def initialize
+    @@board = [[1, 2, 3], [4, 5, 6], [7, 8, 9]]
+    @array_of_number = []
+    @number = nil
+    @@token = nil
+    @turn = true
+    @@tie = nil
+  end
+
+  def creation_of_board
+    3.times do |x|
+      p @@board[x].join(' ')
+    end
+  end
+
+  def change_turn
+    @@token = @turn ? 'X' : 'O'
+  end
+
+  def cases_for_number_selected
+    case @number
+    when 1..3
+      @@board[0][@number - 1] = @@token
+      @array_of_number << @number
+    when 4..6
+      puts @@board[1][@number - 4] = @@token
+      @array_of_number << @number
+    when 7..9
+      puts @@board[2][@number - 7] = @@token
+      @array_of_number << @number
+    end
```

Comment:
![image](https://user-images.githubusercontent.com/26124023/69834169-0d12e780-1239-11ea-9212-1b621d7c6ac0.png)
* [ ] No puts/print and get.chomp outside main.rb file.